core/src/test/scala/integration/kafka/network/DynamicConnectionQuotaTest.scala

```diff
@@ -65,10 +65,6 @@ class DynamicConnectionQuotaTest extends BaseRequestTest {
     }
   }
 
-  override protected def brokerPropertyOverrides(properties: Properties): Unit = {
```

Comment:
This is a minor cleanup: the super function is a no-op so this override is not needed.